src/ecs/commands/updateEnableExecuteCommandFlag.ts

```diff
@@ -38,6 +38,13 @@ export async function updateEnableExecuteCommandFlag(
     const redundantActionMessage = enable
         ? localize('AWS.command.ecs.enableEcsExec.alreadyEnabled', 'ECS Exec is already enabled for this service')
         : localize('AWS.command.ecs.enableEcsExec.alreadyDisabled', 'ECS Exec is already disabled for this service')
+    const updatingServiceMessage = enable
+        ? localize(
+              'AWS.ecs.updateService.enable',
+              'Enabling ECS Exec for service: "{0}". Waiting for deployment...',
```

Comment:
Maybe change `Waiting for deployment...` to `This can take several minutes.` since we aren't listening for a status change to remove the message.

Alternatively I think you can poll for this state, then close the notification on success/fail?